.pylintrc

```diff
@@ -93,6 +93,10 @@ disable=C0103,
         useless-suppression,
         deprecated-pragma,
         use-symbolic-message-instead,
+        too-few-public-methods,
+        too-many-locals,
```

Comment:
Rather than completely disabling this check it might be worth simply increasing the value for [max-locals](https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/too-many-locals.html)...

```
[design]
max-locals = 11
```